app/src/main/java/com/keylesspalace/tusky/util/ThemeUtils.kt

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2017 Andrew Dawson
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+package com.keylesspalace.tusky.util
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.PorterDuff
+import android.graphics.drawable.Drawable
+import android.util.TypedValue
+import androidx.annotation.AttrRes
+import androidx.annotation.ColorInt
+import androidx.appcompat.app.AppCompatDelegate
+
+/**
+ * Provides runtime compatibility to obtain theme information and re-theme views, especially where
+ * the ability to do so is not supported in resource files.
+ */
+object ThemeUtils {
```

Comment:
`ThemeUtils`/`TimeStampUtils` shouldn't be objects but top-level functions.

`object` in Kotlin is similar to a singleton pattern in Java, the class will be instantiated and kept around at runtime, which makes no sense if it does not hold any state.
Top-level functions on the other hand are like static functions in Java (=what they were before the conversion) and thus have less overhead. With an `@file:JvmName` annotation you can choose the name for the class that holds the static functions and if you set it to `ThemeUtils`/`TimeStampUtils` no Java callsites have to be updated. See e.g. `LinkHelper` for that pattern.